go/runtime/host/logger.go

```diff
@@ -77,7 +79,8 @@ func (w RuntimeLogWrapper) processLogLine(line []byte) {
 	// Interpret line as JSON.
 	var m map[string]interface{}
 	if err := json.Unmarshal(line, &m); err != nil {
-		w.logger.Warn("non-JSON log line from runtime", "log_line", string(line), "err", err)
+		// If not valid JSON, forward line as normal log message with local timestamp.
```

Comment:
Should this get it's own changelog entry/commit?

Reply:
Yes, it's already in its own commit, but I'll add a changelog entry and move to its own PR.

Reply:
Moved to #4924.